provisioning/security/tpm-provider-emulator/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/hsm/SecurityProviderTPMEmulator.java

```diff
@@ -102,6 +102,14 @@ public SecurityProviderTPMEmulator(String registrationId) throws SecurityProvide
         srkPublic = createPersistentPrimary(tpm, SRK_PERSISTENT_HANDLE, TPM_RH.OWNER, SRK_TEMPLATE, "SRK");
     }
 
+    public static SecurityProviderTPMEmulator createProviderFromKey(String authenticationKey) throws SecurityProviderException
```

Comment:
Definitely need to add javadocs to explain when you can use this constructor. Users need to know that this constructor is only for cases where the provisioning has already happened.